cmd/root.go

```diff
@@ -62,8 +62,8 @@ func AddCommands(cmd *cobra.Command) {
 		config.BlockCacheSize, "size in layers of meshdb block cache")
 	cmd.PersistentFlags().StringVar(&config.PublishEventsURL, "events-url",
 		config.PublishEventsURL, "publish events to this url; if no url specified no events will be published")
-	cmd.PersistentFlags().BoolVar(&config.Profiler, "profiler",
-		config.Profiler, "enable profiler")
+	cmd.PersistentFlags().StringVar(&config.ProfilerUrl, "profiler-url",
+		config.ProfilerUrl, "send profiler data to certain url, if no url no profiling will be sent")
```

Comment:
We should mention that the URL format should se `http://<IP>:<PORT>`. If we provide `<IP>:<PORT>` then it panics.